packages/elvis/src/components/form/radio-panel.scss

```diff
@@ -0,0 +1,222 @@
+  -webkit-line-clamp: $line-clamp;
+  -webkit-box-orient: vertical;
+}
+
+@mixin square($size) {
+  height: $size;
+  width: $size;
+}
+
+.e-radio-panel-container {
+  --min-width: 150px;
+
+  display: grid;
+  gap: 8px;
+  grid-auto-rows: 1fr; //same height rows
+  place-items: stretch;
+  grid-template-columns: repeat(auto-fill, minmax(var(--min-width), 1fr));
+
+  @include breakpoints.gt-mobile() {
+    gap: 16px;
+    --min-width: 200px;
+
+    &:has(.e-radio-panel--detailed) {
+      --min-width: 250px;
+    }
+  }
+}
+
+.e-radio-panel {
+  --border-color: var(--e-color-border-3);
+  --border-width: 1px;
+
+  align-items: center;
+  border-radius: 100px;
+  box-shadow: inset 0 0 0 var(--border-width) var(--border-color);
+  cursor: pointer;
+  display: grid;
+  gap: 8px;
+  grid-template: auto auto / auto;
+  max-width: 350px;
+  min-width: 150px;
+  padding: 16px 24px;
+
+  @include breakpoints.gt-mobile() {
+    gap: 16px;
+    min-width: 200px;
+    padding: 24px 32px;
+  }
+
+  &:hover,
+  &:has(input:checked) {
+    --border-width: 2px;
+  }
+
+  &:has(input:checked) {
+    --border-color: var(--e-color-border-selected-2);
+  }
+
+  &:has(input:focus-visible) {
+    @include outline.e-outline();
+  }
+
+  .e-radio-panel__icon,
+  .e-radio-panel__icon i {
+    grid-area: 1 / 1 / 2 / 2;
+  }
+
+  .e-radio-panel__icon {
+    @include square(26px);
+    display: grid;
+    justify-self: center;
+    place-items: center;
+
+    @include breakpoints.gt-mobile() {
+      @include square(42px);
+    }
+
+    i {
+      @include square(24px);
+      transition:
+        width 0.2s,
+        height 0.2s;
+
+      @include breakpoints.gt-mobile() {
+        @include square(40px);
+      }
+
+      //hide "non" color icon by default,
+      &:not([class*='color']) {
+        visibility: hidden;
+      }
+
+      //but always show if only one icon
+      &:only-child {
+        visibility: visible;
+      }
+    }
+  }
+
+  &:hover {
+    --border-color: var(--e-color-border-hover-1);
+
+    .e-radio-panel__icon {
+      i {
+        @include square(26px);
+
+        @include breakpoints.gt-mobile() {
+          @include square(42px);
+        }
+
+        //show "non" color icon on hover,
+        &:not([class*='color']) {
+          visibility: visible;
+        }
+        //hide "color" icon on hover, unless it's the only icon available
+        &:not(:only-child)[class*='color'] {
+          visibility: hidden;
+        }
+      }
+    }
+  }
+
+  // Hide default radiostyle
+  input {
+    margin: 0;
+    opacity: 0;
+    outline: none;
+    position: absolute;
+  }
+
+  .e-radio-panel__description,
+  .e-radio-panel__label {
+    margin: 0;
+    padding: 0;
+  }
+
+  .e-radio-panel__description {
+    @include mixins.typography('text-micro-strong');
+    @include limit-lines(2);
+    text-align: center;
+
+    @include breakpoints.gt-mobile() {
+      @include mixins.typography('text-small-strong');
+    }
+  }
+
+  &.e-radio-panel--detailed {
```

Comment:
Optimally this would use flex so that the texts are centered inside the panel, however, this would require a new div to be introduced.

We want it like the screenshot below, we don't have a div that wraps all text, and I therefore can align all text to the icon simultaneously:
<img width="1264" alt="Skjermbilde 2024-07-17 kl  08 42 58" src="https://github.com/user-attachments/assets/57e2994a-a06e-4a59-b32e-c2076322f637">

The current implementation is like this:
![Skjermbilde 2024-07-17 kl  08 45 53](https://github.com/user-attachments/assets/d1437ca6-cf86-4a51-a717-7effa1df45ae)
Notice how the overflowing text goes down, while the "label" stays at the same height aligned with the other panels.

Should we refactor this to use flex? If so what should this wrapper be called? Maybe `e-panel__content` for it to be inline with `e-alert__content`?

Reply:
Or we could remove the `max-height` 🤠